src/core/targeting/youtube.ts

```diff
@@ -39,35 +39,27 @@ const buildAdsConfig = (
 		},
 	};
 
-	if (cmpConsent.ccpa) {
-		const canTarget = !cmpConsent.ccpa.doNotSell;
+	if (cmpConsent.framework === 'ccpa' || cmpConsent.framework === 'aus') {
```

Comment:
Non blocking as I'm sure this works, but just for my understanding, what's the difference between this here and doing `cmpConsent.ccpa || cmpConsent.aus`?

Reply:
No functional difference. 'framework' was added to consentstate to make checking the framework slightly more readable and particular use cases simpler such as switch case statements and when you just want to pass the active framework along rather than having to check the existence of each property for example [here](https://github.com/guardian/frontend/pull/24498/commits/1daffb798617cb6824dd937b0e05b111de0ac83d).

Reply:
Makes sense, thank you